src/SharpRaven.sln

```diff
@@ -81,96 +97,136 @@ Global
-		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}.Debug 4.5|x86.ActiveCfg = Debug 4.5|Any CPU
-		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}.Release 4.0|Any CPU.ActiveCfg = Release 4.0|Any CPU
-		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}.Release 4.0|Any CPU.Build.0 = Release 4.0|Any CPU
-		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}.Release 4.0|Mixed Platforms.ActiveCfg = Release 4.0|Any CPU
-		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}.Release 4.0|Mixed Platforms.Build.0 = Release 4.0|Any CPU
-		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}.Release 4.0|x86.ActiveCfg = Release 4.0|Any CPU
-		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}.Release 4.5|Any CPU.ActiveCfg = Release 4.5|Any CPU
-		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}.Release 4.5|Any CPU.Build.0 = Release 4.5|Any CPU
-		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}.Release 4.5|Mixed Platforms.ActiveCfg = Release 4.5|Any CPU
-		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}.Release 4.5|Mixed Platforms.Build.0 = Release 4.5|Any CPU
-		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}.Release 4.5|x86.ActiveCfg = Release 4.5|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.0|Any CPU.ActiveCfg = Debug 4.0|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.0|Any CPU.Build.0 = Debug 4.0|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.0|Mixed Platforms.ActiveCfg = Debug 4.0|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.0|Mixed Platforms.Build.0 = Debug 4.0|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.0|x86.ActiveCfg = Debug 4.0|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|Any CPU.ActiveCfg = Debug 4.5|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|Any CPU.Build.0 = Debug 4.5|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|Mixed Platforms.ActiveCfg = Debug 4.5|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|Mixed Platforms.Build.0 = Debug 4.5|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|x86.ActiveCfg = Debug 4.5|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|Any CPU.ActiveCfg = Release 4.0|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|Any CPU.Build.0 = Release 4.0|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|Mixed Platforms.ActiveCfg = Release 4.0|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|Mixed Platforms.Build.0 = Release 4.0|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|x86.ActiveCfg = Release 4.0|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|Any CPU.ActiveCfg = Release 4.5|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|Any CPU.Build.0 = Release 4.5|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|Mixed Platforms.ActiveCfg = Release 4.5|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|Mixed Platforms.Build.0 = Release 4.5|Any CPU
-		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|x86.ActiveCfg = Release 4.5|Any CPU
```

Comment:
I'm not looking at a particular commit, just the entire diff of the PR as presented by GitHub and there's two project identifiers affected here: `{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}` and `{156621FC-2C48-4CDF-A368-9347BABE9089}`. It seems like they are just moved further down into the `.sln` file. I'll reference that part in another comment.